packages/flow-core/src/index.ts

```diff
@@ -1,7 +1,52 @@
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/ban-ts-comment */
 /// <reference types="vite/client" />
 
 import { ConfigUtil } from "@ollion/flow-core-config";
 import { version } from "../package.json";
+import { LitElement } from "lit";
```

Comment:
I think barrel files are not handled when doing tree shaking https://vitejs.dev/guide/performance#avoid-barrel-files

We should try enabling `sideEffects` option in package.json and check if Vite can tree shake unused components - https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free

Reply:
We can try publishing a beta version, and check if vite handles this correctly